src/doc/FAQ.md

```diff
@@ -31,7 +31,8 @@ There are many use cases,
 Trakeva Missing Backends
 ------------------------
 
-While configuring Ketrew and trying to start a server it may fail with:
+While configuring Ketrew **(≤ 3.0.0)** and trying to start a server it may fail
```

Comment:
Since it's just the "FAQ" I wanted to wait a couple of versions before cleaning that up.